install.sh

```diff
@@ -417,7 +417,7 @@ user_compiler_flags="${CPPFLAGS:-} ${FFLAGS:-}"
 
 compiler_version=$($FPM_FC --version)
 if [[ $compiler_version == *llvm* ]]; then
-  compiler_flag="-mmlir -allow-assumed-rank -g -Ofast"
+  compiler_flag="-g"
```

Comment:
The flang optimizer bug being worked-around here is now documented in issue #205

Reply:
And workaround added in d17b2cde